### ISSUE TITLE
Live YT video doesn't have any audio tracks, so no audio

### ISSUE THREAD

**Issue by thepragandsensdiary** [Rafostar/clapper#491]
![image](https://github.com/user-attachments/assets/59fda66f-fd7e-4fbd-b9b4-2a625be0e1f1)

When pasting a youtube live URI there's no audio tracks on the list. And the application doesn't reproduce any audio. Just wanna hear lofi hippies :C

**Comment by Rafostar** [comment]
Yes, this is a known issue. In fact its not Clapper issue per-se, but media extractor it uses. Parts of that code are gonna get reworked soon (hopefully fixing this in the process) and likely moved to this repo, so I will just keep this issue here (and probably transfer remaining issues when/if moved).

**Comment by thepragandsensdiary** [comment]
It worked on a web based [invidious instance ](https://inv.nadeko.net/watch?v=jfKfPfyJRdk). So I guess clapper's media extractor is the issue huh. Good luck then~ I love the app design ~

**Comment by Rafostar** [comment]
> It worked on a web based [invidious instance](https://inv.nadeko.net/watch?v=jfKfPfyJRdk)

Temporary workaround until fixed can be playing it from Invidious with replaced URI scheme (https -> invidious) like this: `invidious://inv.nadeko.net/watch?v=jfKfPfyJRdk`

**Comment by Rafostar** [comment]
This seems to work fine now from YT. Once Clapper 0.8.0 reaches Flathub it should just work.
